parm/ioda/bufr2ioda/bufr2ioda_satwind_amv_avhrr.json

```diff
@@ -0,0 +1,17 @@
+{
+  "data_format"      : "bufr_d",
+  "data_type"        : "satwnd",
+  "cycle_type"       : "{{ RUN }}",
+  "cycle_datetime"   : "{{ current_cycle | to_YMDH }}",
+  "dump_directory"   : "{{ DMPDIR }}",
+  "ioda_directory"   : "{{ COM_OBS }}",
+  "subsets"          : [ "NC005080" ],
+  "data_description" : "NC005081 EUMETSAT SATWIND, AVHRR NOAA-15/18/19 IR(LW)", 
```

Comment:
The NC type here and in line 8 (subsets) are not the same. Is this ok?

Reply:
resolved via https://github.com/NOAA-EMC/GDASApp/pull/997/commits/df99b8f4821bac5a90dc1ad0494d35c7df3ea590